tests/unittests/libs/test_database.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+from pony import orm
+
+
+@pytest.fixture
+def fake_jira_issue():
+    return {
+        "issue_key": "TEST-1234",
+        "title": "Test Issue",
+        "url": "https://jira.com",
+        "project": "TEST",
+        "severity": "blocker",
+        "status": "Open",
+        "customer_escaped": False,
+        "date_created": "2024-01-01",
+        "last_updated": "2024-01-01",
+    }
+
+
+@pytest.mark.usefixtures("temp_sqlite_db")
```

Comment:
if all tests in this module need `temp_sqlite_db`, it is easier to have this set on the module:

```
pytestmark = [pytest.mark.temp_sqlite_db]
```